etc/config/c++.amazon.properties

```diff
@@ -3858,6 +3858,12 @@ libs.belleviews.description=A library of C++ views that just works for all basic
 libs.belleviews.versions.trunk.version=trunk
 libs.belleviews.versions.trunk.path=/opt/compiler-explorer/libs/belleviews/trunk/sources
 
+libs.beman_iterator_interface.name=beman_iterator_interface
```

Comment:
same here, if we can do the updates to be `beman.itereator_interface`.

CC: @changkhothuychung @junlarsen - more context here https://github.com/compiler-explorer/infra/pull/1453/files#r1836720720